core/server/api/authentication.js

```diff
@@ -298,6 +299,8 @@ authentication = {
                 }));
             }
 
+            spamPrevention.userLogin.reset(null, options.data.connection + tokenParts.email + 'login');
```

Comment:
Question: this function `extractTokenParts` only extracts the token parts. 
the actual validation happens a little bit down
https://github.com/TryGhost/Ghost/pull/7579/files#diff-25f0b2b0cc338d73a55da195594de403R342

I wonder if we are resetting the tries to early?